packages/cli/src/commands/governance/approve.ts

```diff
@@ -11,37 +11,50 @@ export default class Approve extends BaseCommand {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to approve' }),
     from: Flags.address({ required: true, description: "Approver's address" }),
+    useMultiSig: flags.boolean({
+      required: false,
+      description: 'True means the request will be sent through multisig.',
+    }),
   }
 
-  static examples = ['approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631']
+  static examples = [
+    'approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631',
+    'approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631 --useMultiSig true',
+  ]
 
   async run() {
     const res = this.parse(Approve)
     const account = res.flags.from
+    const useMultiSig = res.flags.useMultiSig
     const id = res.flags.proposalID
     this.kit.defaultAccount = account
     const governance = await this.kit.contracts.getGovernance()
-    const governanceApproverMultiSig = await this.kit.contracts.getMultiSig(
-      await governance.getApprover()
-    )
+    const multiSigAddress = await governance.getApprover()
+    const governanceApproverMultiSig = await this.kit.contracts.getMultiSig(multiSigAddress)
+    const approver = useMultiSig ? multiSigAddress : account
 
     // in case target is queued
     if (await governance.isQueued(id)) {
       await governance.dequeueProposalsIfReady().sendAndWaitForReceipt()
     }
 
     await newCheckBuilder(this)
-      .isApprover(account)
+      .isApprover(approver)
+      .addCheck(
+        `${account} is not multisig signatory`,
+        async () => useMultiSig && !(await governanceApproverMultiSig.isowner(account))
+      )
       .proposalExists(id)
       .addCheck(`${id} not already approved`, async () => !(await governance.isApproved(id)))
       .proposalInStage(id, 'Approval')
       .runChecks()
 
-    const tx = await governance.approve(id)
+    const governanceTx = await governance.approve(id)
     const multiSigTx = await governanceApproverMultiSig.submitOrConfirmTransaction(
       governance.address,
-      tx.txo
+      governanceTx.txo
     )
-    await displaySendTx<any>('approveTx', multiSigTx, {}, 'ProposalApproved')
+    const tx = useMultiSig ? multiSigTx : governanceTx
+    await displaySendTx<any>('approveTx', tx, {}, 'ProposalApproved')
```

Comment:
yes, unfortunately it won't compile without it since the return from the 
multisig.submitOrConfirmTransaction() could be either a string or void, and the displaySendTx<A> requires that A is specified then (otherwise it just assumes its one of those types).  

although it looks like I can make it more specific by just doing displaySendTx<string | void> which seems better